openbmc-init-build-env

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 if [ -z "$DISTRO_DEFAULT" ]; then
-  DISTRO_DEFAULT=rocko
+  DISTRO_DEFAULT=warrior
```

Comment:
Oops. This switches everyone to warrier by default :-)
Generally we do full qualification on a per-platform basis before switching it over and only after all platforms have done their qualification, do we switch the default. I'd leave this as rocko for now but probably switch portwell to warrior and have it lead the effort

Reply:
Of course, the goal of this commit was only for the Travis CI to be able to test it automatically, but at least until we have something that we know for sure is more stable it should stay as rocko